lib/utils/utils.ts

```diff
@@ -142,3 +142,17 @@ export const ms = (): number => {
 export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
+
+export const isPlainObject = (obj: any) => {
```

Comment:
Agreed, some comments explaining what this does and why some of these checks are being performed would be nice because it wasn't obvious to me either.

Reply:
Sure, will do tomorrow !